index.d.ts

```diff
@@ -814,6 +817,10 @@ declare namespace Eris {
   }
 
   // Message
+  interface ActionRow {
+    components: Component[];
```

Comment:
```suggestion
    components: Exclude<Component, ActionRow>[];
```

The above is future proof, or simply just use `Button[]`. ActionRows may not contain another ActionRow.